### DISCUSSION TITLE
Cockpit with latest Ubuntu lts external USB drives issue

### DISCUSSION THREAD

**Discussion by rzmuda** [cockpit-project/cockpit#19649] [Q&A]
I am running the latest cockpit from the app store and latest lts Ubuntu desktop 
For some reason my external usb drives in a dock don't get detected but they are usable in the OS and mounted
How can I make this see them? I need some of the factures that this has?

**Comment by jelly** [comment]
Unless mounted it won't show up in the main overview, but you should have an entry under disks like:

![image](https://github.com/cockpit-project/cockpit/assets/67428/fb5e1645-caf1-4085-a554-4b2803cec524)

**Comment by rzmuda** [comment]
I just checked it only shows 1 of the 2. But the system it self have both running

**Reply by jelly** [reply]
Interesting, we use udisks so maybe it helps to post `udiskctl dump` and see if the disk was detected.

**Reply by ClNo** [reply]
I just posted an issue as I have the same problem related to an USB drive containing 2 SDDs:
USB drive not shown (maybe udisks bug?) #21926
